### PR TITLE
Completely remove libvirt support from leapp

### PR DESCRIPTION
**Pull request by vinzenz**

### CODE REVIEW

File: integration-tests/features/httpd-stateless.feature [hunk 1]
```diff
@@ -1,14 +1,5 @@
 Feature: Redeploy stateless HTTP services
 
-Scenario: Migrate between remote hosts using virt-tar-out
-   Given the local virtual machines:
-         | name       | definition          | ensure_fresh |
-         | app-source | centos6-guest-httpd | no           |
-         | target     | centos7-target      | no           |
-    When app-source is migrated to target as a macrocontainer
```
Comment (ncoghlan) [inline integration-tests/features/httpd-stateless.feature]: This simpler step definition can be reclaimed to refer to the rsync-based migration

File: src/leappto/cli.py [hunk 2]
```diff
@@ -51,7 +51,7 @@
     _set_inheritable = None
 
 # Checking for required permissions
-_REQUIRED_GROUPS = ["vagrant", "libvirt"]
+_REQUIRED_GROUPS = ["vagrant"]
```
Comment (ncoghlan) [inline src/leappto/cli.py]: We shouldn't need `vagrant` any more either, should we?

**Comment by ncoghlan** [comment]
Test failures appear to be due to the switch to using IP addresses for target identification confusing the remote authentication tests for the `--ask-pass` and `--identity` options.

File: src/leappto/cli.py (hunk 2 above)
Comment (vinzenz) [inline src/leappto/cli.py]: @ncoghlan Yes I want to get rid of that too, I am just trying to verify it before ;-)

**Pull request merged**